tests/devapps/DesktopTestApp/MainForm.cs

```diff
@@ -449,6 +455,12 @@ private void TabControl1_Selecting(object sender, TabControlCancelEventArgs e)
 
         private void LoadSettings()
         {
+            if (_publicClientHandler == null)
+            {
```

Comment:
Throw an exception here instead?  the test app is totally messed up if this doesn't happen, and unless someone changes the code this won't occur.  I'd throw InvalidOperationException here.  the debug writeline will get missed.

Reply:
Makes sense. I made this change before making the real update... only try to access the UI after it has been created by moving the code to onload. Will make these changes.